### DISCUSSION TITLE
Issue typing

### DISCUSSION THREAD

**Discussion by Ccarey1233** [rustdesk/rustdesk-server-pro#319] [General]
Good evening everyone I am looking for some help.  I am having no issues accessing another computer remotely however I am unable to type anything on that computer.  Any help would be greatly appreciated thanks!

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/wiki/FAQ#windows-uac
https://github.com/rustdesk/rustdesk/wiki/FAQ#keyboard-translation-modes
